pkg/morph/client/container/client.go

```diff
@@ -89,15 +89,9 @@ type opts struct {
 }
 
 func defaultOpts() *opts {
-	return new(opts)
-}
-
-// TryNotary returns option to enable
-// notary invocation tries.
-func TryNotary() Option {
-	return func(o *opts) {
-		o.staticOpts = append(o.staticOpts, client.TryNotary())
-	}
+	var o = new(opts)
+	o.staticOpts = append(o.staticOpts, client.TryNotary())
```

Comment:
IIRC the client structure is used both for side chain and main chain. Main chain can't be notary-enabled now (see `neofs` contract for example). This may change in the future, Neo 3.6 will include NVB and Conflicts attributes, but this will likely take some time.